bin/run.sh

```diff
@@ -24,16 +24,22 @@ test_file="${2}/${1}.spec.js"
 result_file="${3}/results.json"
 
 # Change xtest to test so all tests are run
-sed -i 's/xtest/test/g' "${test_file}"
-sed -i 's/xit/it/g' "${test_file}"
+if [[ "$OSTYPE" == "darwin"* ]]; then # Mac OS X
```

Comment:
Not sure how cross-platform this is but seem to work fine on Linux Docker and Mac for me... if any objections I suppose we could switch to calling out `uname` or something...

Reply:
I believe OSTYPE works fine 👍